actions/team_actions.test.ts

```diff
@@ -74,27 +74,28 @@ jest.mock('utils/browser_history', () => ({
 describe('Actions.Team', () => {
     const currentChannelId = 'currentChannelId';
 
-    let testStore;
-    beforeEach(async () => {
-        testStore = configureStore({
-            entities: {
-                channels: {
-                    currentChannelId,
-                    manuallyUnread: {},
-                },
+    const initialState = {
+        entities: {
+            channels: {
+                currentChannelId,
+                manuallyUnread: {},
             },
-        });
-    });
+        },
+    };
 
     describe('switchTeam', () => {
         test('should switch to a team by its URL if no team is provided', () => {
+            const testStore = configureStore(initialState);
```

Comment:
(question) out of curiosity, why the change from `beforeEach` to configuring the store in each test? I see all tests configure the store similarly (ie no test needs a different initial state eg)?

Reply:
The before would have worked as expected but i just have to write it like
```
    let testStore: ReturnType<typeof configureStore>;
    const initialState = {
        entities: {
            channels: {
                currentChannelId,
                manuallyUnread: {},
            },
        },
    };
    beforeEach(async () => {
        testStore = configureStore({
            entities: {
                channels: {
                    currentChannelId,
                    manuallyUnread: {},
                },
            },
        });
    });
 ```
 
 But i think its better to have separate store in each test case